### PR TITLE
Fix the branding in CLI windows installer

### PR DESCRIPTION
**Pull request by Sridhar-MS**
- All the branding is moved to Monikers.cs
- Fixed the bundle and msi to use that.

TODO: Do the same for OSx and Debian installers.

Fixes - #2633 

---
This change is [Reviewable](https://reviewable.io/reviews/dotnet/cli/2704)

### CODE REVIEW

File: scripts/dotnet-cli-build/Utils/Monikers.cs [hunk 1]
```diff
@@ -9,6 +9,9 @@ namespace Microsoft.DotNet.Cli.Build
     public class Monikers
     {
         public const string SharedFrameworkName = "Microsoft.NETCore.App";
+        public const string CLISDKBrandName = "Microsoft .NET Core SDK";
+        public const string SharedFxBrandName = "Microsoft .NET Core";
+        public const string SharedHostBrandName = "Microsoft .NET Core Host";
```
Comment (Sridhar-MS) [inline scripts/dotnet-cli-build/Utils/Monikers.cs]: @Petermarcu @richlander @piotrpMSFT Do these names seem ok?

File: packaging/windows/sharedframework/bundle.wxs [hunk 2]
```diff
@@ -24,6 +24,7 @@
     <swid:Tag Regid="microsoft.com" InstallPath="[DOTNETHOME]" />
 
     <Variable Name="DOTNETHOME" Type="string" Value="[ProgramFiles6432Folder]dotnet" bal:Overridable="yes" />
+    <Variable Name="BUNDLEMONIKER" Type="string" Value="$(var.ProductMoniker)" bal:Overridable="yes" />
```
Comment (robmen) [inline packaging/windows/sharedframework/bundle.wxs]: Do you really wan this to be overridable from the command-line?

File: packaging/windows/clisdk/bundle.wxs [hunk 3]
```diff
@@ -24,6 +24,7 @@
     <swid:Tag Regid="microsoft.com" InstallPath="[DOTNETHOME]" />
 
     <Variable Name="DOTNETHOME" Type="string" Value="[ProgramFiles6432Folder]dotnet" bal:Overridable="yes" />
+    <Variable Name="BUNDLEMONIKER" Type="string" Value="$(var.ProductMoniker)" bal:Overridable="yes" />
```
Comment (robmen) [inline packaging/windows/clisdk/bundle.wxs]: Sorry, same here. Seems like something you don't want configurable.

File: packaging/windows/sharedframework/bundle.wxs (hunk 2 above)
Comment (Sridhar-MS) [inline packaging/windows/sharedframework/bundle.wxs]: Copy paste error. Will fix it.

File: packaging/debian/dotnet-nightly-debian_config.json [hunk 4]
```diff
@@ -1,42 +0,0 @@
-{
```
Comment (Sridhar-MS) [inline packaging/debian/dotnet-nightly-debian_config.json]: @brthor FYI. I am deleting this file as discussed.

Comment (brthor) [inline packaging/debian/dotnet-nightly-debian_config.json]: 👍

**Comment by Sridhar-MS** [comment]
@brthor @piotrpMSFT Can you review this?

File: scripts/dotnet-cli-build/Utils/Monikers.cs [hunk 5]
```diff
@@ -9,6 +9,9 @@ namespace Microsoft.DotNet.Cli.Build
     public class Monikers
     {
         public const string SharedFrameworkName = "Microsoft.NETCore.App";
```
Comment (piotrpMSFT) [inline scripts/dotnet-cli-build/Utils/Monikers.cs]: This looks great.

@richlander @blackdwarf @Petermarcu this is the branding that is now plubmed through the CLI.

**Comment by piotrpMSFT** [comment]
I looked at the strings and they LGTM.

**Comment by brthor** [comment]
is the todo still true? Looks like you did deb and pkg.

:shipit:

**Comment by Sridhar-MS** [comment]
yes the TODO is no longer valid

**Pull request merged**